### ISSUE TITLE
Missing form label

### ISSUE THREAD

**Issue by phaldiya** [mbenford/ngTagsInput#680]
**ngTagsInput** create an input element to enter tags, Its Missing form label. Because of that its fail on accessibility complines.
![screenshot at jun 09 13-47-12](https://cloud.githubusercontent.com/assets/8172206/15945736/b357fc62-2e48-11e6-9d23-4ba31910afcd.png)

**Comment by mbenford** [comment]
This issue was selected to be spring-cleaned by a script. More information can be found [here](https://github.com/mbenford/ngTagsInput/issues/790).

If anyone thinks this was a mistake and this issue should be reopened, please leave a message below explaining why. Before doing so, please consider reading the [CONTRIBUTING](https://github.com/mbenford/ngTagsInput/blob/master/CONTRIBUTING.md) file.
